services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/utils/RestPreconditionsUtil.java

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.nodemanager.utils;
+
+import com.futurewei.alcor.common.exception.ParameterNullOrEmptyException;
+import com.futurewei.alcor.common.exception.ParameterUnexpectedValueException;
+import com.futurewei.alcor.nodemanager.entity.NodeInfo;
+import org.thymeleaf.util.StringUtils;
+
+public class RestPreconditionsUtil {
```

Comment:
Should be in AlcorCommonLib

Reply:
I agree that this should be common module. But I could not find it in AlcorCommonLib. Currently, it seems like every micro service has this module in their own package. I will change this part once there is a common module for this.

Reply:
You could move most of those util methods to https://github.com/futurewei-cloud/alcor/tree/master/lib/src/main/java/com/futurewei/alcor/common/utils

Reply:
These classes were moved to https://github.com/futurewei-cloud/alcor/tree/master/lib/src/main/java/com/futurewei/alcor/common/utils/RestPreconditionsUtil.